bun/whatsapp-with-vonage/static/index.html

```diff
@@ -0,0 +1,34 @@
+<!doctype html >
```

Comment:
Please remove the space here <!doctype html >
```suggestion
<!doctype html>
```